draft-ietf-anima-brski-prm.md

```diff
@@ -238,9 +235,11 @@ RER:
 : Registrar Enroll-Request is the CSR of a PER sent to the CA by the domain registrar (in its role as PKI RA).
 
 RVR:
-: Registrar Voucher-Request is a request for a voucher signed by the domain registrar to the MASA.
+: Registrar Voucher-Request is a signature-wrapped voucher-request, signed by the domain registrar that sends it to the MASA.
 It may contain the PVR received from the pledge.
 
+TODO: Section 7.3.2 says "MUST contain ... prior-signed-voucher-request: PVR as received from Registrar-Agent"
```

Comment:
... MUST should be OK for BRSKI-PRM here

Reply:
My understanding is that the **P**VR only exists in BRSKI-PRM; basic BRSKI has the "voucher-request" artifact.

Reply:
In BRSKI , there also exists a [PVR ](https://datatracker.ietf.org/doc/html/rfc8995#name-pledge-requests-voucher-fro) and an  [RVR ](https://datatracker.ietf.org/doc/html/rfc8995#name-registrar-requests-voucher-) with the PVR contained. We just came up with explicit naming for the two artifacts in BRSKI-PRM.